internal/pkg/util/fs/helper.go

```diff
@@ -176,23 +176,33 @@ func MakeTmpFile(basedir, pattern string, mode os.FileMode) (*os.File, error) {
 	return f, nil
 }
 
-// CopyFile copies file to the provided location making sure the resulting
-// file has permission bits set to the mode.
-func CopyFile(from, to string, mode os.FileMode) (err error) {
-	dstFile, err := MakeTmpFile(filepath.Dir(to), "", mode)
+// FileExists simply checks if a path exists.
+func FileExists(path string) (bool, error) {
```

Comment:
Should we keep this one and remove the one from `e2e/internal/e2e/fileutil.go`? I can see e2e using normal packages but not normal packages depending on code from e2e. Maybe actually move both `PathExists()` and `FileExists()`. What do you think? If you agree, maybe open an issue and do it in a different PR to make it easier to review this one?

Reply:
@WestleyK and I chatted and we both think we should move the code from e2e to `helper.go` but in a separate PR, until then @WestleyK is opening an issue.